sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-processing-steps/validate-disbursement-part-time-step.ts

```diff
@@ -38,6 +47,63 @@ export class ValidateDisbursementPartTimeStep
       );
       shouldContinue = false;
     }
+    this.validateCSLPLifetimeMaximumCheckStep(
```

Comment:
The result of this method should be returned, similar to the `stopPartTimeDisbursement` check.
```ts
const lifetimeMaximumReached = await this.validateCSLPLifetimeMaximumCheckStep(
    eCertDisbursement,
    entityManager,
    log,
  );
if (lifetimeMaximumReached) {
  shouldContinue = false;
}
```